tests/runtests.py

```diff
@@ -244,6 +244,13 @@ def setup_collect_tests(start_at, start_after, test_labels=None):
     settings.LOGGING = log_config
     settings.SILENCED_SYSTEM_CHECKS = [
         "fields.W342",  # ForeignKey(unique=True) -> OneToOneField
+        # django.contrib.postgres.fields.CICharField deprecated.
+        "postgres.E001",
```

Comment:
What's `postgres.E001`? 🤔

Reply:
It's caused by `array_field = ArrayField(CITextField(), null=True)` in a test model.

Reply:
`ArrayField` should propagate warnings as a warning, not an error, so I added `postgres.W004`.